packages/components/notifications/content-notification/README.md

```diff
@@ -1,20 +1,20 @@
 # ContentNotification
```

Comment:
The `README` is located under `packages/components/notifications/content-notification`. I think we don't a README under `packages/components/notifications`